### ISSUE TITLE
Write bind_rows.tbl_pb

### ISSUE THREAD

**Issue by NickCH-K** [NickCH-K/pmdplyr#27]
I've tried writing `bind_rows.tbl_pb` a number of times, but I can never get `bind_rows(pibble)` to actually call it. This is the only major uncovered `dplyr` function.

**Comment by philip-khor** [comment]
Happy to take a shot at this, feel free to assign me to this issue.

**Comment by philip-khor** [comment]
The problem is that bind_rows and bind_cols are not actually generic functions: 

```
> library(sloop)
> s3_methods_generic("bind_rows")
# A tibble: 1 x 4
  generic   class  visible source 
  <chr>     <chr>  <lgl>   <chr>  
1 bind_rows tbl_pb TRUE    pmdplyr
Warning message:
In .S3methods(generic.function, class, envir) :
  function 'bind_rows' appears not to be S3 generic; found functions that look like S3 methods
> s3_methods_generic("bind_cols")
# A tibble: 1 x 4
  generic   class  visible source 
  <chr>     <chr>  <lgl>   <chr>  
1 bind_cols tbl_pb TRUE    pmdplyr
Warning message:
In .S3methods(generic.function, class, envir) :
  function 'bind_cols' appears not to be S3 generic; found functions that look like S3 methods
```

tsibble gets around this by implementing methods for cbind and rbind instead, but I'd prefer if we could keep the names bind_cols and bind_rows. tibbletime does not seem to implement either, and simply does `as_tbl_time()` after the call. 

I would recommend not implementing the bind_cols and bind_rows methods as of now. 

See also tidyverse/dplyr#2457

**Comment by NickCH-K** [comment]
Oh, hey, it turns out that all the scoped verb variants aren't generics either! That explains why they aren't being run in the codecov. They're just wrappers for their non-scoped variants. So that's easy and will give a little bit of cleanup.

**Comment by NickCH-K** [comment]
Addressed by d4d87401d964f81b9237a2fa0c0282da53604033
